pagopa/io-app
# fix: [IABT-1474] Avoid clearing contextual help data on logout

## Short description
This PR aims to prevent unintentional clear of the contextual help

## List of changes proposed in this pull request
- change rootReducer initialization

## How to test
1. Open the app 
2. login 
3. check the contextual help from messages home
4. logout
5. login without closing the app
6. check again the contextual help, data should not be missing.
